src/index.html

```diff
@@ -11,7 +11,148 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form action="http://localhost:8080/api" method="post"
+    class="form">
+      <fieldset class="form__group">
+        <legend>Personal information</legend>
+        <div class="form-field">
+          Surname:
+          <input
+            type="text"
+            autocomplete="off"
+          >
+        </div>
+        <div class="form-field">
+          Name:
```

Comment:
Such text should be a label for appropriate input so upon clicking on it correct field is focused